src/progress-indicator/progress-indicator.style.tsx

```diff
@@ -1,24 +1,18 @@
 import styled from "styled-components";
 import { Color } from "../color";
-import { Layout } from "../layout";
 import { MediaQuery } from "../media";
 import { Text } from "../text";
 
 // Units in rem
-export const INDICATOR_BAR_WIDTH_MOBILE = 10;
 export const INDICATOR_BAR_MARGIN_RIGHT_MOBILE = 0.5;
-export const INDICATOR_BAR_FADE_WIDTH_MOBILE = 4;
 
 // =============================================================================
 // STYLE TYPES
 // =============================================================================
-interface FadeProps {
-    backgroundColor: string[];
-    position?: "left" | "right";
-}
 
 interface IndicatorProps {
     highlighted: boolean;
+    isMobile?: boolean;
```

Comment:
this can be reverted as the prop is not used (since the components were split out)